pillow_jxl/JpegXLImagePlugin.py

```diff
@@ -87,7 +87,7 @@ def _save(im, fp, filename, save_all=False):
 
     # default quality is 1
     lossless = info.get("lossless", False)
-    quality = 0 if lossless else 1
+    quality = 0 if lossless else info.get("quality", 1)
```

Comment:
Maybe we can add a mapping from JPEG-Style quality to `FrameDistance` before `jpegxl-rs` fix the mismatch bug?

Reply:
I think you are right and a further refinement could be to use this [JxlEncoderDistanceFromQuality](https://libjxl.readthedocs.io/en/latest/api_encoder.html#_CPPv429JxlEncoderDistanceFromQualityf) function which provides a quality mapping similar to that of jpeg.